crates/tauri-bundler/src/bundle.rs

```diff
@@ -13,6 +13,28 @@ mod settings;
 mod updater_bundle;
 mod windows;
 
+impl From<goblin::error::Error> for crate::error::Error {
```

Comment:
this can be done inside crate::error::Error::BinaryParseError by leverating thiserror `#[from]` attribute